go-controller/pkg/ovn/controller/services/load_balancer.go

```diff
@@ -56,24 +59,28 @@ var protos = []v1.Protocol{
 // - services with NodePort set
 // - services with host-network endpoints
 // - services with ExternalTrafficPolicy=Local
+// - services with InternalTrafficPolicy=Local
 func buildServiceLBConfigs(service *v1.Service, endpointSlices []*discovery.EndpointSlice) (perNodeConfigs []lbConfig, clusterConfigs []lbConfig) {
 	// For each svcPort, determine if it will be applied per-node or cluster-wide
 	for _, svcPort := range service.Spec.Ports {
 		eps := util.GetLbEndpoints(endpointSlices, svcPort)
 
-		// if ExternalTrafficPolicy is local, then we need to do things a bit differently
+		// if ExternalTrafficPolicy or InternalTrafficPolicy is local, then we need to do things a bit differently
```

Comment:
Hmm the doc states if u have etp local then itp local is ignore ?

Reply:
yeah the docs are wrong, I have opened a PR in k8s to fix this: https://github.com/kubernetes/website/pull/32877